st2common/st2common/logging/handlers.py

```diff
@@ -37,7 +38,8 @@ def __init__(self, filename, mode='a', maxBytes=0, backupCount=0, encoding=None,
         format_values = {
             'timestamp': timestamp,
             'ts': isotime_str,
-            'pid': pid
+            'pid': pid,
+            'pack': os.path.abspath(os.path.dirname(sys.argv[2])).split('/')[-2]
```

Comment:
I'm not sure I fully understand how this is supposed to work.

It seems like we parse the pack name from the command line arguments? If so, this doesn't seem robust (not to mention, that it relies on an unstable command line arguments order, etc.).

Is there a safer and more robust way to achieve this?